docs/zh-CN/installing/upgrade.md

```diff
@@ -80,3 +80,12 @@ Doris 可以通过滚动升级的方式，平滑进行升级。建议按照以
 1. 确认新版本的文件部署完成后。逐台重启 FE 和 BE 实例即可。
 2. 建议逐台重启 BE 后，再逐台重启 FE。因为通常 Doris 保证 FE 到 BE 的向后兼容性，即老版本的 FE 可以访问新版本的 BE。但可能不支持老版本的 BE 访问新版本的 FE。
 3. 建议确认前一个实例启动成功后，在重启下一个实例。实例启动成功的标识，请参阅安装部署文档。
+
+## 关于版本回滚
+因为数据库是一个有状态的服务，所以在大多数情况下，Palo/Doris 无法支持版本回滚（版本降级）。在某些情况下，可以支持 3 位或 4 位版本的回滚，但不会支持 2 位版本的回滚。
```

Comment:
Once Doris FE is upgraded and started, it does not support version rollback, regardless of the two-digit version number or the three-digit version number

Have you tried this?

Reply:
In many cases it is possible, but there is currently no clear guarantee. So this is a suggestion